ci/build_py.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+
+################################################################################
+# SETUP - Check environment
+################################################################################
+
+gpuci_logger "Get env"
+env
+
+gpuci_logger "Activate conda env"
+. /opt/conda/etc/profile.d/conda.sh
+conda activate rapids
+
+gpuci_logger "Check versions"
+python --version
+$CC --version
+$CXX --version
+
+gpuci_logger "Check conda environment"
+conda info
+conda config --show-sources
+conda list --show-channel-urls
+
+################################################################################
+# BUILD - Conda package builds (RMM)
+################################################################################
+gpuci_logger "Begin build"
+CPP_FILE_NAME="conda_librmm_build_${BRANCH_NAME}-arc-${ARC}.tar"
+PY_FILE_NAME="conda_rmm_build_${BRANCH_NAME}-arc-${ARC}-py-${PY_VER}.tar"
```

Comment:
These S3 file names should be updated now. Let's use these conventions:

For PRs:

`ci/<REPO_NAME>/pull-request/<PR_NUMBER>/<COMMIT_HASH>/<PACKAGE_NAME>_<PACKAGE_ARCH>.tar`

For branch builds:

`ci/<REPO_NAME>/branch/<BRANCH_NAME>/<COMMIT_HASH>/<PACKAGE_NAME>_<PACKAGE_ARCH>.tar`

For nightly builds:

`nightly/<REPO_NAME>/nightly/<DATE>/<COMMIT_HASH>/<PACKAGE_NAME>_<PACKAGE_ARCH>.tar`

Since the paths are dynamically computed based on environment variables and also vary per build type, I'm thinking it might be best to create a small Golang utility to consolidate this logic. Ideally the Golang utility should inspect the environment variables and print out a file path that can be saved to a bash variable. i.e.:

 ```sh
CPP_FILEPATH=$(golang_tool cpp)
PY_FILEPATH=$(golang_tool python)
```

Reply:
The paths make sense to me. Why use a golang utility though? How would we distribute the utility?

Reply:
> Why use a golang utility though? How would we distribute the utility?

A single binary that's downloaded to a `PATH` directory in our CI images

Reply:
If we want to keep it lightweight, I guess it could also just be a small bash script that's added to our CI images and included on the `PATH`.

Reply:
Okay, I think this makes sense then.

Reply:
Also, @AyodeAwe, let's change the `arc` references to `arch` throughout this PR. I think `arch` is a more proper abbreviation for `architecture`. It also aligns with the [arch](https://man7.org/linux/man-pages/man1/arch.1.html) command, which prints the machine architecture.

Reply:
After some discussions with Sevag and Jordan, I think it makes sense to just add some new functions to the `gpuci_tools` package to facilitate this reusability.